### ISSUE TITLE
After updating to Visual Studio Code 1.6.1 any file is automatically touched upon opening it

### ISSUE THREAD

**Issue by mkjsix** [editorconfig/editorconfig-vscode#81]
This is a copy of this [issue](https://github.com/Microsoft/vscode/issues/13759).

It seems with VSC 1.6.1 the EditorConfig plugin touches any files automatically. How I can disable this behavior?

**Comment by SamVerschueren** [comment]
Duplicate of #57 ?

**Comment by mkjsix** [comment]
Yes, thank you @SamVerschueren,
I followed @jedmao's suggestion:
```
git config core.autocrlf false
git rm --cached -r .
git reset --hard
```
And it looks fine now.

**Comment by jedmao** [comment]
@mkjsix fixed it properly in #116, published as v0.6.1.
